certification/internal/cli/openshift.go

```diff
@@ -37,21 +35,21 @@ type OperatorGroupData struct {
 }
 
 type OpenshiftEngine interface {
-	CreateNamespace(name string, opts OpenshiftOptions, config *rest.Config) (*OpenshiftReport, error)
```

Comment:
I see... so you mean having a separate interface for each resource? I can do that, but these are all openshift-related methods.

Reply:
It's not completely necessary. But is more idiomatic golang.

Reply:
I think I voted the other way when this was initially getting written. 

In fairness, I think I expected a `CreateResource` instead of per-resource method definitions, but I still think that all of them share the same context (rest config, openshift options), so to me, the way it sits here makes sense.

I'm fine going a different direction if we need to, but I don't think this should block this PR from merging. 

Just my 2c. I'm happy to discuss.